README.md

```diff
@@ -1,5 +1,29 @@
-# Momento client-types-php
+# client-sdk-php
 
-:warning: Pre-Alpha :warning:
+:warning: Experimental SDK :warning:
+
+PHP SDK for Momento is experimental and under active development. There could be non-backward compatible changes or
+removal in the future. Please be aware that you may need to update your source code with the current version of the SDK
+when its version gets upgraded.
+---
```

Comment:
not a blocker for this PR but let's get this merged:

https://github.com/momentohq/standards-and-practices/pull/19

and then let's switch this repo over to use it.